app/containers/SignInPage/actions.js

```diff
@@ -0,0 +1,32 @@
+import {
+  REQUEST_SIGN_IN,
```

Comment:
Please follow this rules for actions names: `WHAT_HAPPENS`, `WHAT_HAPPENS_SUCCESS`, `WHAT_HAPPENS_FAILURE`. In your case would be `SIGN_IN`, `SIGN_IN_SUCCESS` and `SIGN_IN_FAILURE`.